Makefile

```diff
@@ -0,0 +1,29 @@
+root = $(realpath $(dir $(realpath $(firstword $(MAKEFILE_LIST)))))
```

Comment:
It is quite a mouth full, but that guaranties that if someone does `make -f /path/to/makefile run` nothing blows up.